tests/framework/builder.py

```diff
@@ -145,7 +145,7 @@ def build(
         with open(config.local_path(), encoding="utf-8") as microvm_config_file:
             microvm_config = json.load(microvm_config_file)
 
-        response = vm.basic_config(
+        vm.basic_config(
```

Comment:
nit: commit message: this looks like a fix to me rather than refactoring.

Reply:
I chose "refactor" since it does not break test bahavior, but it can be also said "fix".

https://www.conventionalcommits.org/en/v1.0.0/
> fix: a commit of the type fix patches a bug in your codebase (this correlates with [PATCH](http://semver.org/#summary) in Semantic Versioning).

This PR already got 2 reviews, so I'll choose "fix" in similar situations. Thanks.